docs/modules/reference/pages/glossary.adoc

```diff
@@ -21,14 +21,18 @@ Auto-discovered nodes:: Nodes that are automatically added to {page-component-ti
 (See xref:operation:provisioning/auto-discovery.adoc[Auto-Discovery].)
 
 Availability:: The percentage of time a node has been detected as being accessible.
-{page-component-title} typically shows availability statistics for the past 24 hours, and calculates this by subtracting the amount of outage time for a node/interface/service.
+{page-component-title} typically shows availability statistics for the past 24 hours; this is calculated by subtracting the amount of outage time for a node, interface, or service.
```

Comment:
I don't think we need to say "for the given time period" since we're talking about a period of 24 hours being shown. If we do want to break this into two sentences, though, maybe the below?

```suggestion
{page-component-title} typically shows availability statistics for the past 24 hours.
This is calculated by subtracting the amount of outage time for a node, interface, or service during the time period.
```

Reply:
I like it.